<!DOCTYPE html>
<html lang="en" class="h-full">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width,initial-scale=1">
    <meta name="turbo-root" content="/v7_manual_rework-preview">
    <meta name="turbo-cache-control" content="no-cache" data-turbo-track="reload" data-track-token="3.11.0.823724251645">

    <!-- See retype.com -->
    <meta name="generator" content="Retype 3.11.0">

    <!-- Primary Meta Tags -->
    <title>Phase 2 sc/snRNA-seq Data Submission</title>
    <meta name="title" content="Phase 2 sc/snRNA-seq Data Submission">
    <meta name="description" content="In HTAN Phase 2, the following files are submitted for single cell/single nuclei RNA-sequencing (sc/snRNA-seq) data:">

    <!-- Canonical -->
    <link rel="canonical" href="https://ncihtan.github.io/v7_manual_rework-preview/data_submission/scrnaseq_data_submission/">

    <!-- Open Graph / Facebook -->
    <meta property="og:type" content="website">
    <meta property="og:url" content="https://ncihtan.github.io/v7_manual_rework-preview/data_submission/scrnaseq_data_submission/">
    <meta property="og:title" content="Phase 2 sc/snRNA-seq Data Submission">
    <meta property="og:description" content="In HTAN Phase 2, the following files are submitted for single cell/single nuclei RNA-sequencing (sc/snRNA-seq) data:">
    <meta property="og:image" content="https://ncihtan.github.io/v7_manual_rework-preview/img/anndata_schema.svg">

    <!-- Twitter -->
    <meta property="twitter:card" content="summary_large_image">
    <meta property="twitter:url" content="https://ncihtan.github.io/v7_manual_rework-preview/data_submission/scrnaseq_data_submission/">
    <meta property="twitter:title" content="Phase 2 sc/snRNA-seq Data Submission">
    <meta property="twitter:description" content="In HTAN Phase 2, the following files are submitted for single cell/single nuclei RNA-sequencing (sc/snRNA-seq) data:">
    <meta property="twitter:image" content="https://ncihtan.github.io/v7_manual_rework-preview/img/anndata_schema.svg">

    <script data-cfasync="false">(function(){var cl=document.documentElement.classList,ls=localStorage.getItem("retype_scheme"),hd=cl.contains("dark"),hl=cl.contains("light"),wm=window.matchMedia&&window.matchMedia("(prefers-color-scheme: dark)").matches;if(ls==="dark"||(!ls&&wm&&!hd&&!hl)){cl.remove("light");cl.add("dark")}else if(ls==="light"||(!ls&&!wm&&!hd&&!hl)){cl.remove("dark");cl.add("light")}})();</script>

    <link href="../../resources/css/retype.css?v=3.11.0.823724251645" rel="stylesheet">

    <script data-cfasync="false" src="../../resources/js/config.js?v=3.11.0.823724251645" data-turbo-eval="false" defer></script>
    <script data-cfasync="false" src="../../resources/js/retype.js?v=3.11.0" data-turbo-eval="false" defer></script>
    <script id="lunr-js" data-cfasync="false" src="../../resources/js/lunr.js?v=3.11.0.823724251645" data-turbo-eval="false" defer></script>
</head>
<body>
    <div id="retype-app" class="relative text-base antialiased text-base-text bg-base-bg font-body">
        <div class="absolute bottom-0 left-0" style="top: 5rem; right: 50%"></div>
    
        <header id="retype-header" class="sticky top-0 z-30 flex w-full h-16 bg-header-bg border-b border-header-border md:h-20">
            <div class="container relative flex items-center justify-between pr-6 grow md:justify-start">
                <!-- Mobile menu button skeleton -->
                <button v-cloak class="skeleton retype-mobile-menu-button flex items-center justify-center shrink-0 overflow-hidden dark:text-white focus:outline-none rounded-full w-10 h-10 ml-3.5 md:hidden"><svg xmlns="http://www.w3.org/2000/svg" class="mb-px shrink-0" width="24" height="24" viewBox="0 0 24 24" role="presentation" style="margin-bottom: 0px;"><g fill="currentColor"><path d="M2 4h20v2H2zM2 11h20v2H2zM2 18h20v2H2z"></path></g></svg></button>
                <div v-cloak id="retype-sidebar-left-toggle-button"></div>
        
                <!-- Logo -->
                <div class="flex items-center justify-between h-full py-2 md:w-75">
                    <div class="flex items-center px-2 md:px-6">
                        <a id="retype-branding-logo" href="../../" class="flex items-center leading-snug text-2xl">
                            <span class="dark:text-white font-bold line-clamp-1 md:line-clamp-2">Project Name</span>
                        </a>
                    </div>
        
                    <span class="hidden h-8 border-r md:inline-block border-base-border"></span>
                </div>
        
                <div class="flex justify-between md:grow">
                    <!-- Top Nav -->
                    <nav id="retype-header-nav" class="hidden md:flex">
                        <ul class="flex flex-col mb-4 md:pl-16 md:mb-0 md:flex-row md:items-center">
                            
                        </ul>
                    </nav>
        
                    <!-- Header Right Skeleton -->
                    <div v-cloak class="flex justify-end grow skeleton">
        
                        <!-- Search input mock -->
                        <div class="relative hidden w-40 lg:block lg:max-w-sm lg:ml-auto">
                            <div class="absolute flex items-center justify-center h-full pl-3 dark:text-dark-300">
                                <svg xmlns="http://www.w3.org/2000/svg" class="icon-base" width="16" height="16" viewBox="0 0 24 24" aria-labelledby="icon" role="presentation" style="margin-bottom: 1px;"><g fill="currentColor" ><path d="M21.71 20.29l-3.68-3.68A8.963 8.963 0 0020 11c0-4.96-4.04-9-9-9s-9 4.04-9 9 4.04 9 9 9c2.12 0 4.07-.74 5.61-1.97l3.68 3.68c.2.19.45.29.71.29s.51-.1.71-.29c.39-.39.39-1.03 0-1.42zM4 11c0-3.86 3.14-7 7-7s7 3.14 7 7c0 1.92-.78 3.66-2.04 4.93-.01.01-.02.01-.02.01-.01.01-.01.01-.01.02A6.98 6.98 0 0111 18c-3.86 0-7-3.14-7-7z" ></path></g></svg>
                            </div>
                            <input class="w-full h-10 placeholder-search-placeholder transition-colors duration-200 ease-in bg-search-bg border border-transparent rounded md:text-sm hover:border-search-border-hover focus:outline-none focus:border-search-border-focus" style="padding: 0.625rem 0.75rem 0.625rem 2rem" type="text" placeholder="Search">
                        </div>
        
                        <!-- Mobile search button -->
                        <div class="flex items-center justify-center w-10 h-10 lg:hidden">
                            <svg xmlns="http://www.w3.org/2000/svg" class="shrink-0 icon-base" width="20" height="20" viewBox="0 0 24 24" aria-labelledby="icon" role="presentation" style="margin-bottom: 0px;"><g fill="currentColor" ><path d="M21.71 20.29l-3.68-3.68A8.963 8.963 0 0020 11c0-4.96-4.04-9-9-9s-9 4.04-9 9 4.04 9 9 9c2.12 0 4.07-.74 5.61-1.97l3.68 3.68c.2.19.45.29.71.29s.51-.1.71-.29c.39-.39.39-1.03 0-1.42zM4 11c0-3.86 3.14-7 7-7s7 3.14 7 7c0 1.92-.78 3.66-2.04 4.93-.01.01-.02.01-.02.01-.01.01-.01.01-.01.02A6.98 6.98 0 0111 18c-3.86 0-7-3.14-7-7z" ></path></g></svg>
                        </div>
        
                        <!-- Dark mode switch placeholder -->
                        <div class="w-10 h-10 lg:ml-2"></div>
        
                        <!-- History button -->
                        <div class="flex items-center justify-center w-10 h-10" style="margin-right: -0.625rem;">
                            <svg xmlns="http://www.w3.org/2000/svg" class="shrink-0 icon-base" width="22" height="22" viewBox="0 0 24 24" aria-labelledby="icon" role="presentation" style="margin-bottom: 0px;"><g fill="currentColor" ><g ><path d="M12.01 6.01c-.55 0-1 .45-1 1V12a1 1 0 00.4.8l3 2.22a.985.985 0 001.39-.2.996.996 0 00-.21-1.4l-2.6-1.92V7.01c.02-.55-.43-1-.98-1z"></path><path d="M12.01 1.91c-5.33 0-9.69 4.16-10.05 9.4l-.29-.26a.997.997 0 10-1.34 1.48l1.97 1.79c.19.17.43.26.67.26s.48-.09.67-.26l1.97-1.79a.997.997 0 10-1.34-1.48l-.31.28c.34-4.14 3.82-7.41 8.05-7.41 4.46 0 8.08 3.63 8.08 8.09s-3.63 8.08-8.08 8.08c-2.18 0-4.22-.85-5.75-2.4a.996.996 0 10-1.42 1.4 10.02 10.02 0 007.17 2.99c5.56 0 10.08-4.52 10.08-10.08.01-5.56-4.52-10.09-10.08-10.09z"></path></g></g></svg>
                        </div>
                    </div>
        
                    <div v-cloak class="flex justify-end grow">
                        <div id="retype-mobile-search-button"></div>
                        <doc-search-desktop></doc-search-desktop>
        
                        <doc-theme-switch class="lg:ml-2"></doc-theme-switch>
                        <doc-history></doc-history>
                    </div>
                </div>
            </div>
        </header>
    
    
        <div id="retype-container" class="container relative flex bg-white">
            <!-- Sidebar Skeleton -->
            <div v-cloak class="fixed flex flex-col shrink-0 duration-300 ease-in-out bg-sidebar-left-bg border-sidebar-left-border sidebar top-20 w-75 border-r h-screen md:sticky transition-transform skeleton">
            
                <div class="flex items-center h-16 px-6">
                    <input class="w-full h-8 px-3 py-2 transition-colors duration-200 ease-linear bg-filter-bg border border-filter-border rounded shadow-none text-sm focus:outline-none focus:border-filter-border-focus" type="text" placeholder="Filter">
                </div>
            
                <div class="pl-6 mt-1 mb-4">
                    <div class="w-32 h-3 mb-4 bg-skeleton-bg rounded-full loading"></div>
                    <div class="w-48 h-3 mb-4 bg-skeleton-bg rounded-full loading"></div>
                    <div class="w-40 h-3 mb-4 bg-skeleton-bg rounded-full loading"></div>
                    <div class="w-32 h-3 mb-4 bg-skeleton-bg rounded-full loading"></div>
                    <div class="w-48 h-3 mb-4 bg-skeleton-bg rounded-full loading"></div>
                    <div class="w-40 h-3 mb-4 bg-skeleton-bg rounded-full loading"></div>
                </div>
            
                <div class="shrink-0 mt-auto bg-transparent dark:border-base-border">
                    <a class="flex items-center justify-center flex-nowrap h-16 text-gray-350 dark:text-dark-400 hover:text-gray-600 dark:hover:text-dark-300 transition-colors duration-150 ease-in docs-powered-by" target="_blank" href="https://retype.com/" rel="noopener">
                        <span class="text-xs whitespace-nowrap">Powered by</span>
                        <svg xmlns="http://www.w3.org/2000/svg" class="ml-2" fill="currentColor" width="96" height="20" overflow="visible"><path d="M0 0v20h13.59V0H0zm11.15 17.54H2.44V2.46h8.71v15.08zM15.8 20h2.44V4.67L15.8 2.22zM20.45 6.89V20h2.44V9.34z"/><g><path d="M40.16 8.44c0 1.49-.59 2.45-1.75 2.88l2.34 3.32h-2.53l-2.04-2.96h-1.43v2.96h-2.06V5.36h3.5c1.43 0 2.46.24 3.07.73s.9 1.27.9 2.35zm-2.48 1.1c.26-.23.38-.59.38-1.09 0-.5-.13-.84-.4-1.03s-.73-.28-1.39-.28h-1.54v2.75h1.5c.72 0 1.2-.12 1.45-.35zM51.56 5.36V7.2h-4.59v1.91h4.13v1.76h-4.13v1.92h4.74v1.83h-6.79V5.36h6.64zM60.09 7.15v7.48h-2.06V7.15h-2.61V5.36h7.28v1.79h-2.61zM70.81 14.64h-2.06v-3.66l-3.19-5.61h2.23l1.99 3.45 1.99-3.45H74l-3.19 5.61v3.66zM83.99 6.19c.65.55.97 1.4.97 2.55s-.33 1.98-1 2.51-1.68.8-3.04.8h-1.23v2.59h-2.06V5.36h3.26c1.42 0 2.45.28 3.1.83zm-1.51 3.65c.25-.28.37-.69.37-1.22s-.16-.92-.48-1.14c-.32-.23-.82-.34-1.5-.34H79.7v3.12h1.38c.68 0 1.15-.14 1.4-.42zM95.85 5.36V7.2h-4.59v1.91h4.13v1.76h-4.13v1.92H96v1.83h-6.79V5.36h6.64z"/></g></svg>
                    </a>
                </div>
            </div>
            
            <!-- Sidebar component -->
            <doc-sidebar v-cloak>
                <template #sidebar-footer>
                    <div class="shrink-0 mt-auto border-t md:bg-transparent md:border-none dark:border-base-border">
            
                        <a class="flex items-center justify-center flex-nowrap h-16 text-gray-350 dark:text-dark-400 hover:text-gray-600 dark:hover:text-dark-300 transition-colors duration-150 ease-in docs-powered-by" target="_blank" href="https://retype.com/" rel="noopener">
                            <span class="text-xs whitespace-nowrap">Powered by</span>
                            <svg xmlns="http://www.w3.org/2000/svg" class="ml-2" fill="currentColor" width="96" height="20" overflow="visible"><path d="M0 0v20h13.59V0H0zm11.15 17.54H2.44V2.46h8.71v15.08zM15.8 20h2.44V4.67L15.8 2.22zM20.45 6.89V20h2.44V9.34z"/><g><path d="M40.16 8.44c0 1.49-.59 2.45-1.75 2.88l2.34 3.32h-2.53l-2.04-2.96h-1.43v2.96h-2.06V5.36h3.5c1.43 0 2.46.24 3.07.73s.9 1.27.9 2.35zm-2.48 1.1c.26-.23.38-.59.38-1.09 0-.5-.13-.84-.4-1.03s-.73-.28-1.39-.28h-1.54v2.75h1.5c.72 0 1.2-.12 1.45-.35zM51.56 5.36V7.2h-4.59v1.91h4.13v1.76h-4.13v1.92h4.74v1.83h-6.79V5.36h6.64zM60.09 7.15v7.48h-2.06V7.15h-2.61V5.36h7.28v1.79h-2.61zM70.81 14.64h-2.06v-3.66l-3.19-5.61h2.23l1.99 3.45 1.99-3.45H74l-3.19 5.61v3.66zM83.99 6.19c.65.55.97 1.4.97 2.55s-.33 1.98-1 2.51-1.68.8-3.04.8h-1.23v2.59h-2.06V5.36h3.26c1.42 0 2.45.28 3.1.83zm-1.51 3.65c.25-.28.37-.69.37-1.22s-.16-.92-.48-1.14c-.32-.23-.82-.34-1.5-.34H79.7v3.12h1.38c.68 0 1.15-.14 1.4-.42zM95.85 5.36V7.2h-4.59v1.91h4.13v1.76h-4.13v1.92H96v1.83h-6.79V5.36h6.64z"/></g></svg>
                        </a>
                    </div>
                </template>
            </doc-sidebar>
    
            <div class="grow min-w-0 bg-body-bg">
                <!-- Render "toolbar" template here on api pages --><!-- Render page content -->
                <div class="flex">
                    <div id="retype-main" class="min-w-0 p-4 grow md:px-16">
                        <main class="relative pb-12 lg:pt-2">
                            <div class="retype-markdown" id="retype-content">
                                <!-- Rendered if sidebar right is enabled -->
                                <div id="retype-sidebar-right-toggle"></div>
                                <!-- Page content  -->
<doc-anchor-target id="phase-2-scsnrna-seq-data-submission" class="break-words">
    <h1>
        <doc-anchor-trigger class="header-anchor-trigger" to="#phase-2-scsnrna-seq-data-submission">#</doc-anchor-trigger>
        <span>Phase 2 sc/snRNA-seq Data Submission</span>
    </h1>
</doc-anchor-target>
<doc-anchor-target id="overview">
    <h2>
        <doc-anchor-trigger class="header-anchor-trigger" to="#overview">#</doc-anchor-trigger>
        <span>Overview</span>
    </h2>
</doc-anchor-target>
<p>In HTAN Phase 2, the following files are submitted for single cell/single nuclei RNA-sequencing (sc/snRNA-seq) data:</p>
<div class="table-wrapper scrollbar overflow-hidden">
<table class="comfortable">
<thead>
<tr>
<th>Level</th>
<th>Data Type</th>
<th>Example Files</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>raw sequence data</td>
<td>fastq, unaligned bam</td>
</tr>
<tr>
<td>2</td>
<td>aligned sequence data</td>
<td>bam</td>
</tr>
<tr>
<td>3_4</td>
<td>sample level summary information, e.g. cell annotations, t-SNE/UMAP coordinates, etc.</td>
<td>h5ad</td>
</tr>
</tbody>
</table>
</div>
<p>Metadata requirements are documented in the HTAN Data Model <a href="https://htan2-data-model.readthedocs.io/en/main/">readthedocs</a> pages. This part of the manual describes <strong>file requirements</strong> for level 3_4 h5ad files.</p>
<doc-anchor-target id="htans-h5ad-requirements">
    <h2>
        <doc-anchor-trigger class="header-anchor-trigger" to="#htans-h5ad-requirements">#</doc-anchor-trigger>
        <span>HTAN&#x27;s h5ad Requirements</span>
    </h2>
</doc-anchor-target>
<p>HTAN Centers are encouraged to reference the <a href="https://docs.google.com/document/d/1XjDLWulYWhnfZrGCg-0_Jh93ytIp3p_01ZrTyymTjoU/edit?usp=sharing">sc/snRNA-seq RFC</a> for additional details. The HTAN h5ad (AnnData 0.10) requirements are modeled after CELLxGENE&#x27;s requirements.  They also include three attributes developed by the Human Cell Atlas (HCA). Please see the <doc-anchor-trigger to="#background-h5ad-files-cellxgene-human-cell-atlas">Background</doc-anchor-trigger> section below for more information about h5ad (AnnData 0.10) files, CELLxGENE and the HCA.</p>
<doc-anchor-target id="required-file-attributes">
    <h3>
        <doc-anchor-trigger class="header-anchor-trigger" to="#required-file-attributes">#</doc-anchor-trigger>
        <span>Required File Attributes</span>
    </h3>
</doc-anchor-target>
<p>Similar to CELLxGENE&#x27;s <a href="https://cellxgene.cziscience.com/docs/032__Contribute%20and%20Publish%20Data">Dataset Requirements</a>, level 3_4 sc/snRNA-seq h5ad files must contain the following attributes. Please see <a href="https://github.com/ncihtan/h5ad/blob/main/exemplars/HTAN_h5ad_exemplar_2025_03_03.h5ad">HTAN_h5ad_exemplar_2025_03_03.h5ad</a> for an example file which meets these requirements.</p>
<div class="table-wrapper scrollbar overflow-hidden">
<table class="comfortable">
<thead>
<tr>
<th>Attribute</th>
<th>Ontology/Version</th>
<th>Comments/Examples</th>
</tr>
</thead>
<tbody>
<tr>
<td>var.index, raw.var.index</td>
<td><a href="https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/5.2.0/schema.md#required-gene-annotations">Human reference GRCh38.p14 (GENCODE v44/Ensembl 110)</a></td>
<td>ENSEMBL IDs.  For example:  ENSG00000107566.</td>
</tr>
<tr>
<td>var.gene_is_filtered, raw.var.gene_id_filtered</td>
<td></td>
<td>no genes filtered in raw data; if gene is filtered in normalized data, count is set to 0 and gene_is_filtered set to 1.</td>
</tr>
<tr>
<td>obs.organism_ontology_term_id</td>
<td><a href="https://www.ncbi.nlm.nih.gov/taxonomy">NCBITaxon</a></td>
<td>Set to NCBITaxon:9606 for human.</td>
</tr>
<tr>
<td>obs.donor_id</td>
<td></td>
<td>Set to the HTAN Participant ID, e.g. HTA201_1.</td>
</tr>
<tr>
<td>obs.sample_id</td>
<td></td>
<td>Set to the HTAN Biospecimen ID, e.g. HTA201_1_B.</td>
</tr>
<tr>
<td>obs.development_stage_ontology_term_id</td>
<td><a href="https://www.ebi.ac.uk/ols4/ontologies/HsapDv">Human Development Stages (HsapDv)</a></td>
<td>use <a href="https://docs.google.com/document/d/1SsHZweG_kqerCAPNbQF7gQHNBDRqOsNZWzaWXZIKwTE/edit?usp=sharing">HCA recommended terms</a> (p.22)</td>
</tr>
<tr>
<td>obs.sex_ontology_term_id</td>
<td><a href="https://www.ebi.ac.uk/ols4/ontologies/pato">Phenotype and Trait Ontology (PATO)</a></td>
<td>Use <a href="https://cellxgene.cziscience.com/docs/032__Contribute%20and%20Publish%20Data#dataset-requirements">CELLxGENE Requirements</a> PATO:0000384 for male, PATO:0000383 for female, or unknown if unavailable.</td>
</tr>
<tr>
<td>obs.self_reported_ethnicity_term_id</td>
<td><a href="https://www.ebi.ac.uk/ols4/ontologies/hancestro">Human Ancestry Ontology (HANCESTRO)</a></td>
<td>Use <a href="https://cellxgene.cziscience.com/docs/032__Contribute%20and%20Publish%20Data#dataset-requirements">CELLxGENE Requirements</a>. HANCESTRO multiple comma-separated terms may be used if more than one ethnicity is reported. If information is unavailable, use unknown. Example:  HANCESTRO_0568. Note that CELLxGENE specifically excludes certain HANCESTRO categories.  See <a href="https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/5.2.0/schema.md#self_reported_ethnicity_ontology_term_id">full details</a>.</td>
</tr>
<tr>
<td>obs.disease_ontology_term_id</td>
<td><a href="https://www.ebi.ac.uk/ols4/ontologies/mondo">Mondo Disease Ontology</a></td>
<td></td>
</tr>
<tr>
<td>obs.tissue_type</td>
<td>CELLxGENE</td>
<td>Use <a href="https://cellxgene.cziscience.com/docs/032__Contribute%20and%20Publish%20Data#dataset-requirements">CELLxGENE Requirements</a>, Permitted values are restricted to: tissue, organoid, or cell culture.</td>
</tr>
<tr>
<td>obs.tissue_ontology_term_id</td>
<td><a href="https://www.ebi.ac.uk/ols4/ontologies/uberon">Uber Anatomy Ontology (UBERON)</a></td>
<td></td>
</tr>
<tr>
<td>obs.cell_type_ontology_term_id</td>
<td><a href="https://www.ebi.ac.uk/ols4/ontologies/cl">Cell Ontology (CL)</a></td>
<td></td>
</tr>
<tr>
<td>obs.assay_ontology_term_id</td>
<td><a href="https://www.ebi.ac.uk/ols4/ontologies/efo">Experimental Factor Ontology (EFO)</a></td>
<td>Use <a href="https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/5.2.0/schema.md#assay_ontology_term_id">CELLxGENE Requirements</a></td>
</tr>
<tr>
<td>obs.suspension_type</td>
<td>CELLxGENE</td>
<td>Use <a href="https://cellxgene.cziscience.com/docs/032__Contribute%20and%20Publish%20Data#dataset-requirements">CELLxGENE Requirements</a>. Permitted values are restricted to: cell, nucleus, na.</td>
</tr>
<tr>
<td>obs.is_primary_data</td>
<td>CELLxGENE.  Used to indicate if this is the canonical data set (True), or data is being reused from another source (False).</td>
<td>Use <a href="https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/5.2.0/schema.md#is_primary_data">CELLxGENE Requirements</a>.  Permitted values are restricted to True or False.</td>
</tr>
<tr>
<td>obs.cell_enrichment</td>
<td>Human Cell Atlas:  “Specifies the cell types targeted for enrichment or depletion beyond the selection of live cells.“</td>
<td>CL term, followed by + or -.  If no enrichment. Then use CL:00000000. For example, enrichment for fibroblasts would be CL:0000057+</td>
</tr>
<tr>
<td>obs.intron_inclusion</td>
<td>Human Cell Atlas: “Were introns included during read counting in the alignment process?”</td>
<td>Permitted values are:  yes, no</td>
</tr>
<tr>
<td>obs.author_cell_type</td>
<td>Human Cell Atlas: “Encoding of author intuition of cellular annotation in the dataset.”</td>
<td>Free text</td>
</tr>
<tr>
<td><a href="https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/5.2.0/schema.md#x_suffix">obsm.X_(suffix)</a></td>
<td>CELLxGENE:  embeddings of at least two dimensions, e.g.  tSNE, UMAP, PCA, spatial coordinates</td>
<td>use <a href="https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/5.2.0/schema.md#x_suffix">CELLxGENE terms</a> for suffix (e.g. umap, tsne, pca)</td>
</tr>
</tbody>
</table>
</div>
<doc-anchor-target id="htan-h5ad-file-validation">
    <h3>
        <doc-anchor-trigger class="header-anchor-trigger" to="#htan-h5ad-file-validation">#</doc-anchor-trigger>
        <span>HTAN h5ad File Validation</span>
    </h3>
</doc-anchor-target>
<p>The HTAN Data Coordinating Center (DCC) has released a PyPi package called <a href="https://pypi.org/project/HTAN-h5ad-validate/">HTAN-h5ad-validator</a> with which Centers can validate their sc/snRNA-seq h5ad files. Sage Bionetworks will run the validator on sc/snRNA-seq h5ad files submitted to Synapse.</p>
<doc-anchor-target id="background-h5ad-files-cellxgene-human-cell-atlas">
    <h2>
        <doc-anchor-trigger class="header-anchor-trigger" to="#background-h5ad-files-cellxgene-human-cell-atlas">#</doc-anchor-trigger>
        <span>Background: h5ad files, CELLxGENE, Human Cell Atlas</span>
    </h2>
</doc-anchor-target>
<doc-anchor-target id="h5ad-anndata-010-brief-overview">
    <h3>
        <doc-anchor-trigger class="header-anchor-trigger" to="#h5ad-anndata-010-brief-overview">#</doc-anchor-trigger>
        <span>h5ad (AnnData 0.10) brief overview</span>
    </h3>
</doc-anchor-target>
<p>Please see <a href="https://anndata.readthedocs.io/en/latest/index.html">AnnData’s documentation</a> for a more detailed description of the AnnData object.</p>
<p><figure class="content-center">
    <img src="../../img/anndata_schema.svg" alt="From https://raw.githubusercontent.com/scverse/anndata/main/docs/_static/img/anndata_schema.svg" class="rounded-image-rounded border-image-border border-image-border-width" />
    <figcaption class="caption">From https://raw.githubusercontent.com/scverse/anndata/main/docs/_static/img/anndata_schema.svg</figcaption>
</figure>
</p>
<p>For HTAN’s purposes, the following parts of the AnnData object are of interest:</p>
<ul>
<li>.X - a matrix with counts where rows are cells and columns are genes.</li>
<li>var - a matrix with gene information (e.g. gene name, gene_is_filtered).</li>
<li>obs - a matrix with cell-level information.</li>
<li>obsm - one or more numpy ndarrays with cell embeddings.</li>
</ul>
<p>CELLxGENE requires that raw data are submitted.  Normalized data may also be submitted.</p>
<doc-anchor-target id="cellxgene">
    <h3>
        <doc-anchor-trigger class="header-anchor-trigger" to="#cellxgene">#</doc-anchor-trigger>
        <span>CELLxGENE</span>
    </h3>
</doc-anchor-target>
<p>The HTAN DCC submits sc/snRNA-seq data to <a href="https://cellxgene.cziscience.com">CELLxGENE</a>, a tool developed by the Chan Zuckerberg Initiative (CZI) to visualize and explore single cell and spatial data. The DCC submits data to CellxGene in h5ad (AnnData 0.10) format. CELLxGENE’s schema requires:</p>
<ul>
<li>use of Ensembl gene IDs.</li>
<li>a specific genome reference and annotation version.</li>
<li>specific h5ad (AnnData 0.10) attributes.</li>
<li>use of specific ontologies for many of the required attributes (i.e. cell ontology).</li>
</ul>
<p>The HTAN requirements for h5ad files are modeled after CELLxGENE&#x27;s <a href="https://cellxgene.cziscience.com/docs/032__Contribute%20and%20Publish%20Data">Dataset Requirements</a>.</p>
<doc-anchor-target id="human-cell-atlas-hca">
    <h3>
        <doc-anchor-trigger class="header-anchor-trigger" to="#human-cell-atlas-hca">#</doc-anchor-trigger>
        <span>Human Cell Atlas (HCA)</span>
    </h3>
</doc-anchor-target>
<p>The Human Cell Atlas (HCA) is a large repository of single cell data from healthy subjects.  It provides <a href="https://docs.google.com/document/d/1SsHZweG_kqerCAPNbQF7gQHNBDRqOsNZWzaWXZIKwTE/edit?usp=sharing">standards for single-cell data submission</a> which adopt most of the CELLxGENE schema, but also include additional fields.  Aligning HTAN data with CELLxGENE will potentially facilitate data integration with other consortia such as the HCA.  The HTAN requirements include three HCA attributes in addition to CELLxGENE required attributes.</p>

                                
                                <!-- Required only on API pages -->
                                <doc-toolbar-member-filter-no-results></doc-toolbar-member-filter-no-results>
                            </div>
                            <footer id="retype-content-footer" class="clear-both">
                            
                                <nav id="retype-nextprev" class="print:hidden flex mt-14">
                                    <div class="w-1/2">
                                        <a class="px-5 py-4 h-full flex items-center break-normal font-medium text-body-link border border-base-border hover:border-base-border-hover rounded-l-lg transition-colors duration-150 relative hover:z-5" href="../../data_submission/clin_data_submission/">
                                            <svg xmlns="http://www.w3.org/2000/svg" class="mr-3" width="24" height="24" viewBox="0 0 24 24" fill="currentColor" overflow="visible"><path d="M19 11H7.41l5.29-5.29a.996.996 0 10-1.41-1.41l-7 7a1 1 0 000 1.42l7 7a1.024 1.024 0 001.42-.01.996.996 0 000-1.41L7.41 13H19c.55 0 1-.45 1-1s-.45-1-1-1z" /><path fill="none" d="M0 0h24v24H0z" /></svg>
                                            <span>
                                                <span class="block text-xs font-normal text-base-text-muted">Previous</span>
                                                <span class="block mt-1">Phase 2 Clinical Data Submission</span>
                                            </span>
                                        </a>
                                    </div>
                            
                                    <div class="w-1/2">
                                        <a class="px-5 py-4 -mx-px h-full flex items-center justify-end break-normal font-medium text-body-link border border-base-border hover:border-base-border-hover rounded-r-lg transition-colors duration-150 relative hover:z-5" href="../../addtnl_info/tool_protocol/">
                                            <span>
                                                <span class="block text-xs font-normal text-right text-base-text-muted">Next</span>
                                                <span class="block mt-1">Submitting Tools and Protocols</span>
                                            </span>
                                            <svg xmlns="http://www.w3.org/2000/svg" class="ml-3" width="24" height="24" viewBox="0 0 24 24" fill="currentColor" overflow="visible"><path d="M19.92 12.38a1 1 0 00-.22-1.09l-7-7a.996.996 0 10-1.41 1.41l5.3 5.3H5c-.55 0-1 .45-1 1s.45 1 1 1h11.59l-5.29 5.29a.996.996 0 000 1.41c.19.2.44.3.7.3s.51-.1.71-.29l7-7c.09-.09.16-.21.21-.33z" /><path fill="none" d="M0 0h24v24H0z" /></svg>
                                        </a>
                                    </div>
                                </nav>
                            </footer>
                        </main>
                
                        <div id="retype-page-footer" class="print:border-none border-t border-base-border pt-6 mb-8">
                            <footer class="flex flex-wrap items-center justify-between print:justify-center">
                                <div id="retype-footer-links" class="print:hidden">
                                    <ul class="flex flex-wrap items-center text-sm">
                                    </ul>
                                </div>
                                <div id="retype-copyright" class="print:justify-center py-2 text-footer-text font-footer-link-weight text-sm leading-relaxed"></div>
                            </footer>
                        </div>
                    </div>
                
                    <!-- Rendered if sidebar right is enabled -->
                    <!-- Sidebar right skeleton-->
                    <div v-cloak class="fixed top-0 bottom-0 right-0 translate-x-full bg-sidebar-right-bg border-sidebar-right-border lg:sticky lg:border-l lg:shrink-0 lg:pt-6 lg:transform-none sm:w-1/2 lg:w-64 lg:z-0 md:w-104 sidebar-right skeleton">
                        <div class="pl-5">
                            <div class="w-32 h-3 mb-4 bg-skeleton-bg rounded-full loading"></div>
                            <div class="w-48 h-3 mb-4 bg-skeleton-bg rounded-full loading"></div>
                            <div class="w-40 h-3 mb-4 bg-skeleton-bg rounded-full loading"></div>
                        </div>
                    </div>
                
                    <!-- User should be able to hide sidebar right -->
                    <doc-sidebar-right v-cloak></doc-sidebar-right>
                </div>

            </div>
        </div>
    
        <doc-search-mobile></doc-search-mobile>
        <doc-back-to-top></doc-back-to-top>
    </div>


    <div id="retype-overlay-target"></div>

    <script data-cfasync="false">window.__DOCS__ = { "title": "Phase 2 sc/snRNA-seq Data Submission", level: 2, icon: "file", hasPrism: false, hasMermaid: false, hasMath: false, tocDepth: 23 }</script>
</body>
</html>
