<!DOCTYPE html>
<html lang="en" class="h-full">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width,initial-scale=1">
    <meta name="turbo-root" content="/htan_missing_manual-preview">
    <meta name="turbo-cache-control" content="no-cache" data-turbo-track="reload" data-track-token="3.11.0.823724600190">

    <!-- See retype.com -->
    <meta name="generator" content="Retype 3.11.0">

    <!-- Primary Meta Tags -->
    <title>File Standards</title>
    <meta name="title" content="File Standards">
    <meta name="description" content="The HTAN Data Model includes requirements for Assay Files. These include:">

    <!-- Canonical -->
    <link rel="canonical" href="https://ncihtan.github.io/htan_missing_manual-preview/data_model/file_standards/">

    <!-- Open Graph / Facebook -->
    <meta property="og:type" content="website">
    <meta property="og:url" content="https://ncihtan.github.io/htan_missing_manual-preview/data_model/file_standards/">
    <meta property="og:title" content="File Standards">
    <meta property="og:description" content="The HTAN Data Model includes requirements for Assay Files. These include:">
    <meta property="og:image" content="https://ncihtan.github.io/htan_missing_manual-preview/img/sequencing_data_levels.svg">

    <!-- Twitter -->
    <meta property="twitter:card" content="summary_large_image">
    <meta property="twitter:url" content="https://ncihtan.github.io/htan_missing_manual-preview/data_model/file_standards/">
    <meta property="twitter:title" content="File Standards">
    <meta property="twitter:description" content="The HTAN Data Model includes requirements for Assay Files. These include:">
    <meta property="twitter:image" content="https://ncihtan.github.io/htan_missing_manual-preview/img/sequencing_data_levels.svg">

    <script data-cfasync="false">(function(){var cl=document.documentElement.classList,ls=localStorage.getItem("retype_scheme"),hd=cl.contains("dark"),hl=cl.contains("light"),wm=window.matchMedia&&window.matchMedia("(prefers-color-scheme: dark)").matches;if(ls==="dark"||(!ls&&wm&&!hd&&!hl)){cl.remove("light");cl.add("dark")}else if(ls==="light"||(!ls&&!wm&&!hd&&!hl)){cl.remove("dark");cl.add("light")}})();</script>

    <link href="../../resources/css/retype.css?v=3.11.0.823724600190" rel="stylesheet">

    <script data-cfasync="false" src="../../resources/js/config.js?v=3.11.0.823724600190" data-turbo-eval="false" defer></script>
    <script data-cfasync="false" src="../../resources/js/retype.js?v=3.11.0" data-turbo-eval="false" defer></script>
    <script id="lunr-js" data-cfasync="false" src="../../resources/js/lunr.js?v=3.11.0.823724600190" data-turbo-eval="false" defer></script>
</head>
<body>
    <div id="retype-app" class="relative text-base antialiased text-base-text bg-base-bg font-body">
        <div class="absolute bottom-0 left-0" style="top: 5rem; right: 50%"></div>
    
        <header id="retype-header" class="sticky top-0 z-30 flex w-full h-16 bg-header-bg border-b border-header-border md:h-20">
            <div class="container relative flex items-center justify-between pr-6 grow md:justify-start">
                <!-- Mobile menu button skeleton -->
                <button v-cloak class="skeleton retype-mobile-menu-button flex items-center justify-center shrink-0 overflow-hidden dark:text-white focus:outline-none rounded-full w-10 h-10 ml-3.5 md:hidden"><svg xmlns="http://www.w3.org/2000/svg" class="mb-px shrink-0" width="24" height="24" viewBox="0 0 24 24" role="presentation" style="margin-bottom: 0px;"><g fill="currentColor"><path d="M2 4h20v2H2zM2 11h20v2H2zM2 18h20v2H2z"></path></g></svg></button>
                <div v-cloak id="retype-sidebar-left-toggle-button"></div>
        
                <!-- Logo -->
                <div class="flex items-center justify-between h-full py-2 md:w-75">
                    <div class="flex items-center px-2 md:px-6">
                        <a id="retype-branding-logo" href="../../" class="flex items-center leading-snug text-2xl">
                            <span class="dark:text-white font-bold line-clamp-1 md:line-clamp-2">Project Name</span>
                        </a>
                    </div>
        
                    <span class="hidden h-8 border-r md:inline-block border-base-border"></span>
                </div>
        
                <div class="flex justify-between md:grow">
                    <!-- Top Nav -->
                    <nav id="retype-header-nav" class="hidden md:flex">
                        <ul class="flex flex-col mb-4 md:pl-16 md:mb-0 md:flex-row md:items-center">
                            
                        </ul>
                    </nav>
        
                    <!-- Header Right Skeleton -->
                    <div v-cloak class="flex justify-end grow skeleton">
        
                        <!-- Search input mock -->
                        <div class="relative hidden w-40 lg:block lg:max-w-sm lg:ml-auto">
                            <div class="absolute flex items-center justify-center h-full pl-3 dark:text-dark-300">
                                <svg xmlns="http://www.w3.org/2000/svg" class="icon-base" width="16" height="16" viewBox="0 0 24 24" aria-labelledby="icon" role="presentation" style="margin-bottom: 1px;"><g fill="currentColor" ><path d="M21.71 20.29l-3.68-3.68A8.963 8.963 0 0020 11c0-4.96-4.04-9-9-9s-9 4.04-9 9 4.04 9 9 9c2.12 0 4.07-.74 5.61-1.97l3.68 3.68c.2.19.45.29.71.29s.51-.1.71-.29c.39-.39.39-1.03 0-1.42zM4 11c0-3.86 3.14-7 7-7s7 3.14 7 7c0 1.92-.78 3.66-2.04 4.93-.01.01-.02.01-.02.01-.01.01-.01.01-.01.02A6.98 6.98 0 0111 18c-3.86 0-7-3.14-7-7z" ></path></g></svg>
                            </div>
                            <input class="w-full h-10 placeholder-search-placeholder transition-colors duration-200 ease-in bg-search-bg border border-transparent rounded md:text-sm hover:border-search-border-hover focus:outline-none focus:border-search-border-focus" style="padding: 0.625rem 0.75rem 0.625rem 2rem" type="text" placeholder="Search">
                        </div>
        
                        <!-- Mobile search button -->
                        <div class="flex items-center justify-center w-10 h-10 lg:hidden">
                            <svg xmlns="http://www.w3.org/2000/svg" class="shrink-0 icon-base" width="20" height="20" viewBox="0 0 24 24" aria-labelledby="icon" role="presentation" style="margin-bottom: 0px;"><g fill="currentColor" ><path d="M21.71 20.29l-3.68-3.68A8.963 8.963 0 0020 11c0-4.96-4.04-9-9-9s-9 4.04-9 9 4.04 9 9 9c2.12 0 4.07-.74 5.61-1.97l3.68 3.68c.2.19.45.29.71.29s.51-.1.71-.29c.39-.39.39-1.03 0-1.42zM4 11c0-3.86 3.14-7 7-7s7 3.14 7 7c0 1.92-.78 3.66-2.04 4.93-.01.01-.02.01-.02.01-.01.01-.01.01-.01.02A6.98 6.98 0 0111 18c-3.86 0-7-3.14-7-7z" ></path></g></svg>
                        </div>
        
                        <!-- Dark mode switch placeholder -->
                        <div class="w-10 h-10 lg:ml-2"></div>
        
                        <!-- History button -->
                        <div class="flex items-center justify-center w-10 h-10" style="margin-right: -0.625rem;">
                            <svg xmlns="http://www.w3.org/2000/svg" class="shrink-0 icon-base" width="22" height="22" viewBox="0 0 24 24" aria-labelledby="icon" role="presentation" style="margin-bottom: 0px;"><g fill="currentColor" ><g ><path d="M12.01 6.01c-.55 0-1 .45-1 1V12a1 1 0 00.4.8l3 2.22a.985.985 0 001.39-.2.996.996 0 00-.21-1.4l-2.6-1.92V7.01c.02-.55-.43-1-.98-1z"></path><path d="M12.01 1.91c-5.33 0-9.69 4.16-10.05 9.4l-.29-.26a.997.997 0 10-1.34 1.48l1.97 1.79c.19.17.43.26.67.26s.48-.09.67-.26l1.97-1.79a.997.997 0 10-1.34-1.48l-.31.28c.34-4.14 3.82-7.41 8.05-7.41 4.46 0 8.08 3.63 8.08 8.09s-3.63 8.08-8.08 8.08c-2.18 0-4.22-.85-5.75-2.4a.996.996 0 10-1.42 1.4 10.02 10.02 0 007.17 2.99c5.56 0 10.08-4.52 10.08-10.08.01-5.56-4.52-10.09-10.08-10.09z"></path></g></g></svg>
                        </div>
                    </div>
        
                    <div v-cloak class="flex justify-end grow">
                        <div id="retype-mobile-search-button"></div>
                        <doc-search-desktop></doc-search-desktop>
        
                        <doc-theme-switch class="lg:ml-2"></doc-theme-switch>
                        <doc-history></doc-history>
                    </div>
                </div>
            </div>
        </header>
    
    
        <div id="retype-container" class="container relative flex bg-white">
            <!-- Sidebar Skeleton -->
            <div v-cloak class="fixed flex flex-col shrink-0 duration-300 ease-in-out bg-sidebar-left-bg border-sidebar-left-border sidebar top-20 w-75 border-r h-screen md:sticky transition-transform skeleton">
            
                <div class="flex items-center h-16 px-6">
                    <input class="w-full h-8 px-3 py-2 transition-colors duration-200 ease-linear bg-filter-bg border border-filter-border rounded shadow-none text-sm focus:outline-none focus:border-filter-border-focus" type="text" placeholder="Filter">
                </div>
            
                <div class="pl-6 mt-1 mb-4">
                    <div class="w-32 h-3 mb-4 bg-skeleton-bg rounded-full loading"></div>
                    <div class="w-48 h-3 mb-4 bg-skeleton-bg rounded-full loading"></div>
                    <div class="w-40 h-3 mb-4 bg-skeleton-bg rounded-full loading"></div>
                    <div class="w-32 h-3 mb-4 bg-skeleton-bg rounded-full loading"></div>
                    <div class="w-48 h-3 mb-4 bg-skeleton-bg rounded-full loading"></div>
                    <div class="w-40 h-3 mb-4 bg-skeleton-bg rounded-full loading"></div>
                </div>
            
                <div class="shrink-0 mt-auto bg-transparent dark:border-base-border">
                    <a class="flex items-center justify-center flex-nowrap h-16 text-gray-350 dark:text-dark-400 hover:text-gray-600 dark:hover:text-dark-300 transition-colors duration-150 ease-in docs-powered-by" target="_blank" href="https://retype.com/" rel="noopener">
                        <span class="text-xs whitespace-nowrap">Powered by</span>
                        <svg xmlns="http://www.w3.org/2000/svg" class="ml-2" fill="currentColor" width="96" height="20" overflow="visible"><path d="M0 0v20h13.59V0H0zm11.15 17.54H2.44V2.46h8.71v15.08zM15.8 20h2.44V4.67L15.8 2.22zM20.45 6.89V20h2.44V9.34z"/><g><path d="M40.16 8.44c0 1.49-.59 2.45-1.75 2.88l2.34 3.32h-2.53l-2.04-2.96h-1.43v2.96h-2.06V5.36h3.5c1.43 0 2.46.24 3.07.73s.9 1.27.9 2.35zm-2.48 1.1c.26-.23.38-.59.38-1.09 0-.5-.13-.84-.4-1.03s-.73-.28-1.39-.28h-1.54v2.75h1.5c.72 0 1.2-.12 1.45-.35zM51.56 5.36V7.2h-4.59v1.91h4.13v1.76h-4.13v1.92h4.74v1.83h-6.79V5.36h6.64zM60.09 7.15v7.48h-2.06V7.15h-2.61V5.36h7.28v1.79h-2.61zM70.81 14.64h-2.06v-3.66l-3.19-5.61h2.23l1.99 3.45 1.99-3.45H74l-3.19 5.61v3.66zM83.99 6.19c.65.55.97 1.4.97 2.55s-.33 1.98-1 2.51-1.68.8-3.04.8h-1.23v2.59h-2.06V5.36h3.26c1.42 0 2.45.28 3.1.83zm-1.51 3.65c.25-.28.37-.69.37-1.22s-.16-.92-.48-1.14c-.32-.23-.82-.34-1.5-.34H79.7v3.12h1.38c.68 0 1.15-.14 1.4-.42zM95.85 5.36V7.2h-4.59v1.91h4.13v1.76h-4.13v1.92H96v1.83h-6.79V5.36h6.64z"/></g></svg>
                    </a>
                </div>
            </div>
            
            <!-- Sidebar component -->
            <doc-sidebar v-cloak>
                <template #sidebar-footer>
                    <div class="shrink-0 mt-auto border-t md:bg-transparent md:border-none dark:border-base-border">
            
                        <a class="flex items-center justify-center flex-nowrap h-16 text-gray-350 dark:text-dark-400 hover:text-gray-600 dark:hover:text-dark-300 transition-colors duration-150 ease-in docs-powered-by" target="_blank" href="https://retype.com/" rel="noopener">
                            <span class="text-xs whitespace-nowrap">Powered by</span>
                            <svg xmlns="http://www.w3.org/2000/svg" class="ml-2" fill="currentColor" width="96" height="20" overflow="visible"><path d="M0 0v20h13.59V0H0zm11.15 17.54H2.44V2.46h8.71v15.08zM15.8 20h2.44V4.67L15.8 2.22zM20.45 6.89V20h2.44V9.34z"/><g><path d="M40.16 8.44c0 1.49-.59 2.45-1.75 2.88l2.34 3.32h-2.53l-2.04-2.96h-1.43v2.96h-2.06V5.36h3.5c1.43 0 2.46.24 3.07.73s.9 1.27.9 2.35zm-2.48 1.1c.26-.23.38-.59.38-1.09 0-.5-.13-.84-.4-1.03s-.73-.28-1.39-.28h-1.54v2.75h1.5c.72 0 1.2-.12 1.45-.35zM51.56 5.36V7.2h-4.59v1.91h4.13v1.76h-4.13v1.92h4.74v1.83h-6.79V5.36h6.64zM60.09 7.15v7.48h-2.06V7.15h-2.61V5.36h7.28v1.79h-2.61zM70.81 14.64h-2.06v-3.66l-3.19-5.61h2.23l1.99 3.45 1.99-3.45H74l-3.19 5.61v3.66zM83.99 6.19c.65.55.97 1.4.97 2.55s-.33 1.98-1 2.51-1.68.8-3.04.8h-1.23v2.59h-2.06V5.36h3.26c1.42 0 2.45.28 3.1.83zm-1.51 3.65c.25-.28.37-.69.37-1.22s-.16-.92-.48-1.14c-.32-.23-.82-.34-1.5-.34H79.7v3.12h1.38c.68 0 1.15-.14 1.4-.42zM95.85 5.36V7.2h-4.59v1.91h4.13v1.76h-4.13v1.92H96v1.83h-6.79V5.36h6.64z"/></g></svg>
                        </a>
                    </div>
                </template>
            </doc-sidebar>
    
            <div class="grow min-w-0 bg-body-bg">
                <!-- Render "toolbar" template here on api pages --><!-- Render page content -->
                <div class="flex">
                    <div id="retype-main" class="min-w-0 p-4 grow md:px-16">
                        <main class="relative pb-12 lg:pt-2">
                            <div class="retype-markdown" id="retype-content">
                                <!-- Rendered if sidebar right is enabled -->
                                <div id="retype-sidebar-right-toggle"></div>
                                <!-- Page content  -->
<doc-anchor-target id="file-standards" class="break-words">
    <h1>
        <doc-anchor-trigger class="header-anchor-trigger" to="#file-standards">#</doc-anchor-trigger>
        <span>File Standards</span>
    </h1>
</doc-anchor-target>
<p>The HTAN Data Model includes requirements for Assay Files. These include:</p>
<ol>
<li>How Data Files are organized (<doc-anchor-trigger to="#assay-data-levels">Assay Data Levels</doc-anchor-trigger>);</li>
<li>What File Formats are accepted (<doc-anchor-trigger to="#accepted-file-formats">Accepted File Formats</doc-anchor-trigger>); and</li>
<li>Specific Assay or Assay Module Requirements (<doc-anchor-trigger to="#specific-assayassay-module-requirements">Specific Requirements</doc-anchor-trigger>).</li>
</ol>
<doc-anchor-target id="assay-data-levels">
    <h2>
        <doc-anchor-trigger class="header-anchor-trigger" to="#assay-data-levels">#</doc-anchor-trigger>
        <span>Assay Data Levels</span>
    </h2>
</doc-anchor-target>
<p>In both the HTAN Phase 1 and Phase 2 Data Models, assay data is divided into levels. Each assay type has levels progressing from raw data to more processed data.  (See Figure 1) For Sequencing data, the HTAN Model Data levels help distinguish data as <a href="../../data_access/introduction/">open access versus access-controlled</a>.  For example, level 1 and 2 sequencing data are access-controlled data.</p>
<doc-panel id="figure-1-htan-sequencing-data-levels-expand-panel-to-see-figure" collapsed>
    <template #title>Figure 1. HTAN Sequencing Data levels (Expand Panel to see Figure.)</template>
<p><figure class="content-center">
    <img src="../../img/sequencing_data_levels.svg" alt="Figure 1. HTAN Sequencing Data levels" class="rounded-image-rounded border-image-border border-image-border-width" />
    <figcaption class="caption">Figure 1. HTAN Sequencing Data levels</figcaption>
</figure>
</p>
</doc-panel>
<doc-panel id="phase-2-assay-data-levels" collapsed>
    <template #title>Phase 2 Assay Data Levels</template>
<doc-anchor-target id="phase-2-assay-data-levels">
    <h3>
        <doc-anchor-trigger class="header-anchor-trigger" to="#phase-2-assay-data-levels">#</doc-anchor-trigger>
        <span>Phase 2 Assay Data Levels</span>
    </h3>
</doc-anchor-target>
<p>The Phase 2 Data Model is still being developed. This page will be updated as these updates occur. The following are the currently supported assays, organized alphabetically.</p>
<div class="table-wrapper scrollbar overflow-hidden">
<table class="comfortable">
<thead>
<tr>
<th>Assay Type</th>
<th>Level</th>
<th>Definition</th>
<th>Example Data</th>
<th>Notes</th>
</tr>
</thead>
<tbody>
<tr>
<td>DigitalPathology</td>
<td>2</td>
<td>Imaging data compiled into a single file format which is compatible with <a href="https://www.openmicroscopy.org/bio-formats/">Bio-Formats</a> or <a href="https://openslide.org/api/python/">openslide python</a>.</td>
<td>.ome-tiff, .qptiff, .svs, .tif, .dcm, .ndpi. .vms, .vmu, .scn, .mrxs, .tiff, .svslide, .bit, .czi</td>
<td>currently, there is only one file level for DigitalPathology</td>
</tr>
<tr>
<td>MultiplexedMicroscopy</td>
<td>2</td>
<td>Imaging data compiled into a single file format, preferably a tiled and pyramidal OME-TIFF. Accompanied by a csv file containing channel metadata.</td>
<td>image: ome-tiff; channel metadata: csv</td>
<td>there is no level 1 for MultiplexMicroscopy</td>
</tr>
<tr>
<td></td>
<td>3</td>
<td>Segmentation mask.</td>
<td>ome-tiff</td>
<td></td>
</tr>
<tr>
<td></td>
<td>4</td>
<td>An object-by-feature table (typically cell-by-marker) generated from the segmentation mask and image.</td>
<td>csv, h5ad</td>
<td></td>
</tr>
<tr>
<td>scRNA-seq</td>
<td>1</td>
<td>raw sequence data.</td>
<td>fastq, fastq.gz</td>
<td></td>
</tr>
<tr>
<td></td>
<td>2</td>
<td>aligned sequence data.</td>
<td>bam, cram</td>
<td></td>
</tr>
<tr>
<td></td>
<td>3_4</td>
<td>sample level summary information, e.g. cell annotations, t-SNE/UMAP coordinates, etc.</td>
<td>h5ad</td>
<td>level 3_4 files have specific format and content requirements. Please see <doc-anchor-trigger to="#specific-assayassay-module-requirements">Specific Assay/Assay Module Requirements</doc-anchor-trigger>.</td>
</tr>
<tr>
<td>SpatialOmics</td>
<td>1</td>
<td>Optional level for submission of controlled-access experiment data.</td>
<td>compressed archive (.tar.gz, .zip) containing raw data (e.g. FASTQs, BAMS)</td>
<td>Submission of level 1 is <strong>not required</strong></td>
</tr>
<tr>
<td></td>
<td>3</td>
<td>A compressed archive (e.g., .tar.gz, .zip) containing platform-specific output files. Should include segmentation outputs, raw or normalized matrices, and any relevant vendor JSONs, manifest files, or images.</td>
<td>compressed archive (.tar.gz, .zip)</td>
<td>level 3 bundles <strong>cannot contain controlled-access data</strong> such as FASTQs or BAMs. There is no level 2 for SpatialOmics.</td>
</tr>
<tr>
<td></td>
<td>Panel</td>
<td>Panel information if the assay uses a targeted sequencing or protein panel.</td>
<td>csv</td>
<td></td>
</tr>
<tr>
<td></td>
<td>4</td>
<td>A harmonized output file for downstream analysis.</td>
<td>AnnData h5ad or seurat compatible RDS</td>
<td>level 4 is optional. A HTAN-standardized format for these files (similar to level 3_4 scRNA-seq data) has not been developed, but may exist in the future.</td>
</tr>
<tr>
<td>WES</td>
<td>1</td>
<td>Raw data</td>
<td>fastq, fastq.gz</td>
<td></td>
</tr>
<tr>
<td></td>
<td>2</td>
<td>Aligned primary data</td>
<td>bam, cram</td>
<td></td>
</tr>
<tr>
<td></td>
<td>3</td>
<td>Derived biomolecular data</td>
<td>vcf, vcf.gz</td>
<td></td>
</tr>
</tbody>
</table>
</div>
</doc-panel>
<doc-panel id="phase-1-assay-data-levels" collapsed>
    <template #title>Phase 1 Assay Data Levels</template>
<doc-anchor-target id="phase-1-assay-data-levels">
    <h3>
        <doc-anchor-trigger class="header-anchor-trigger" to="#phase-1-assay-data-levels">#</doc-anchor-trigger>
        <span>Phase 1 Assay Data Levels</span>
    </h3>
</doc-anchor-target>
<p>Phase 1 data was divided into 4 levels for most assay types. Exceptions include:</p>
<ul>
<li>some assays did not have a level 1 (RPPA) or level 1 data was typically not collected (Imaging), but higher levels exist;</li>
<li>some assays did not have higher levels such as scDNA-seq and scmC-seq (only levels 1 and 2), Bulk DNA, Bulk RNA, HiC and Bulk Methylation Sequencing (only levels 1-3);</li>
<li>spatial transcriptomics assays had platform-specific data levels which deviated from the traditional 4 levels. Specifically,
<ul>
<li>some platforms have only one experiment level instead of multiple file levels (10X Xenium ISS and Nanostring CosMx SMI); and</li>
<li>some platforms have additional files (auxiliary or annotation metadata) which were not a part of the typical 4 level system (e.g. 10X Visium, Nanostring GeoMx DSP).</li>
</ul>
</li>
</ul>
<p>Please see the Table below for more information.</p>
<div class="table-wrapper scrollbar overflow-hidden">
<table class="comfortable">
<thead>
<tr>
<th>Assay Type</th>
<th>Level</th>
<th>Definition</th>
<th>Example Data</th>
<th>Notes</th>
</tr>
</thead>
<tbody>
<tr>
<td>Electron Microscopy</td>
<td>1</td>
<td>Raw electron microscopy data as one TIFF file per plane for a 3D image stack or per tile for a 2D large area montage.</td>
<td>tiff</td>
<td></td>
</tr>
<tr>
<td></td>
<td>2</td>
<td>Processed electron microscopy data as one OME-TIFF image per plane or montage.</td>
<td>ome-tiff</td>
<td></td>
</tr>
<tr>
<td></td>
<td>3</td>
<td>Segmented electron microscopy data.</td>
<td>am, tiff</td>
<td></td>
</tr>
<tr>
<td></td>
<td>4</td>
<td>Movies or other derived files from electron microscopy data.</td>
<td>mp4, csv</td>
<td></td>
</tr>
<tr>
<td>Imaging</td>
<td>1</td>
<td>Raw imaging data requiring tiling, stitching, illumination correction, registration or other pre-processing.</td>
<td>svs, tiff</td>
<td>usually not submitted</td>
</tr>
<tr>
<td></td>
<td>2</td>
<td>Imaging data compiled into a single file format, preferably a tiled and pyramidal OME-TIFF. Accompanied by a csv file containing channel metadata.</td>
<td>ome-tiff + csv</td>
<td></td>
</tr>
<tr>
<td></td>
<td>3</td>
<td>Segmentation mask, Validated channel metadata, QC checked image.</td>
<td>ome-tiff</td>
<td></td>
</tr>
<tr>
<td></td>
<td>4</td>
<td>An object-by-feature table (typically cell-by-marker) generated from the segmentation mask and image.</td>
<td>csv</td>
<td></td>
</tr>
<tr>
<td>Mass Spectrometry</td>
<td>1</td>
<td>Raw spectral data.</td>
<td>mz5,dta,ms2,ms1,mzXML,mzML,mzData</td>
<td></td>
</tr>
<tr>
<td></td>
<td>2</td>
<td>Spectrum match peaks.</td>
<td>Peptide spectrum match (PSM) in csv format</td>
<td></td>
</tr>
<tr>
<td></td>
<td>3</td>
<td>Peptide Group information</td>
<td>Combined peptide spectrum as csv or tsv files</td>
<td></td>
</tr>
<tr>
<td></td>
<td>4</td>
<td>Protein Abundance.</td>
<td>csv, tsv</td>
<td></td>
</tr>
<tr>
<td>RPPA</td>
<td>2</td>
<td>Primary data. Array based protemics.</td>
<td>csv</td>
<td>(No level 1 for RPPA)</td>
</tr>
<tr>
<td></td>
<td>3</td>
<td>Intra-batch normalized intensities.</td>
<td>txt,csv</td>
<td></td>
</tr>
<tr>
<td></td>
<td>4</td>
<td>Intra-batch corrected intensities.</td>
<td>txt</td>
<td></td>
</tr>
<tr>
<td>Sequencing</td>
<td>1</td>
<td>Raw data</td>
<td>FASTQs, unaligned BAMs</td>
<td></td>
</tr>
<tr>
<td></td>
<td>2</td>
<td>Aligned primary data</td>
<td>Aligned BAMs</td>
<td></td>
</tr>
<tr>
<td></td>
<td>3</td>
<td>Derived biomolecular data</td>
<td>Gene expression matrix files, VCFs, etc.</td>
<td>no level 3 for scDNA-seq and scmC-seq</td>
</tr>
<tr>
<td></td>
<td>4</td>
<td>Sample level summary data.</td>
<td>t-SNE plot coordinates, etc.</td>
<td>no level 4 for scDNA-seq, scmC-seq, Bulk DNA-seq, Bulk RNA-seq, HiC-Seq, Bulk Methylation-Seq</td>
</tr>
<tr>
<td>Spatial Transcriptomics, Nanostring CosMx SMI</td>
<td>Experiment</td>
<td>RNA and Protein Panel assays applied as part of Nanostring CosMx Spatial Molecular Imager (SMI)</td>
<td>experiment bundle submitted instead of multiple file levels.</td>
<td></td>
</tr>
<tr>
<td>Spatial Transcriptomics, Nanostring GeoMx DSP</td>
<td>1</td>
<td>Files contain raw data output from the NanoString GeoMx DSP Pipeline. These can include RCC or DCC Files.</td>
<td>rcc, dcc</td>
<td></td>
</tr>
<tr>
<td></td>
<td>3</td>
<td>Files contain processed data from the NanoString GeoMx DSP Pipeline. This level depends on GeoMx Level 1 and Imaging Level 2.</td>
<td>csv</td>
<td>Associated ome-tiff images submitted using the Imaging Level 2 manifest.</td>
</tr>
<tr>
<td></td>
<td>DCC ROI Segment Annotation</td>
<td>GeoMx ROI and Segment Metadata Attributes. The assayed biospecimen should be reported one per row with the associated ROI coordinates.</td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td>RCC ROI Segment Annotation</td>
<td>GeoMx ROI and Segment Metadata Attributes. The assayed biospecimen should be reported one per row with the associated ROI coordinates.</td>
<td></td>
<td></td>
</tr>
<tr>
<td>Spatial Transcriptomics, Slide-Seq</td>
<td>1</td>
<td>Raw sequencing files for the Slide-seq assay.</td>
<td>tar.gz (FASTQ)</td>
<td></td>
</tr>
<tr>
<td></td>
<td>2</td>
<td>Aligned sequencing files and QC for the Slide-seq assay.</td>
<td>BAM</td>
<td></td>
</tr>
<tr>
<td></td>
<td>3</td>
<td>Gene matrices with features and barcodes for Slide-seq as well as spatial information (bead location files).</td>
<td>tsv, mtx.gz</td>
<td></td>
</tr>
<tr>
<td>Spatial Transcriptomics, 10x Visium</td>
<td>1</td>
<td>Files contain raw RNA-seq data associated with spot/slide data.</td>
<td>FASTQ</td>
<td></td>
</tr>
<tr>
<td></td>
<td>2</td>
<td>Alignment workflows downstream of Spatial Transcriptomics RNA-seq Level 1.</td>
<td>Aligned BAMS</td>
<td></td>
</tr>
<tr>
<td></td>
<td>3</td>
<td>Processed data files based on Spatial Transcriptomics RNA-seq Level 2 and Spatial Transcriptomics Auxiliary files.</td>
<td>Barcodes, features, filtered and unfiltered matrices</td>
<td></td>
</tr>
<tr>
<td></td>
<td>4</td>
<td>Processed data files based on Spatial Transcriptomics RNA-seq Level 3.</td>
<td>Clustering, t-SNE coordinates, h5ad, RDS</td>
<td></td>
</tr>
<tr>
<td></td>
<td>Auxiliary</td>
<td>Auxiliary data associated with spot/slide analysis (aligned Images, quality control files, etc) from Spatial Transcriptomics. Additional data such as imaging, qc, and json scale factors.</td>
<td>TIFF, JPG, PNG, JSON, HTML, etc.</td>
<td></td>
</tr>
<tr>
<td>Spatial Transcriptomics, 10X Xenium ISS</td>
<td>Experiment</td>
<td>All data pertaining to the 10X Genomics Xenium In-Situ Hybridization experiment</td>
<td>experiment bundle</td>
<td>experiment bundle submitted instead of multiple file levels.</td>
</tr>
</tbody>
</table>
</div>
</doc-panel>
<doc-panel id="phase-2-vs-phase-1-assay-data-levels" collapsed>
    <template #title>Phase 2 vs Phase 1 Assay Data Levels</template>
<doc-anchor-target id="phase-2-vs-phase-1-assay-data-levels">
    <h3>
        <doc-anchor-trigger class="header-anchor-trigger" to="#phase-2-vs-phase-1-assay-data-levels">#</doc-anchor-trigger>
        <span>Phase 2 vs Phase 1 Assay Data Levels</span>
    </h3>
</doc-anchor-target>
<p>As outlined in the <a href="../overview/">Data Model Introduction</a>, the Phase 1 and Phase 2 HTAN Data Models Differ.  In terms of assay data levels, currently, the following differences exist.</p>
<div class="table-wrapper scrollbar overflow-hidden">
<table class="comfortable">
<thead>
<tr>
<th>Assay Modules</th>
<th>Example Assays</th>
<th>Phase 1 Levels</th>
<th>Phase 2 Levels</th>
<th>Change</th>
</tr>
</thead>
<tbody>
<tr>
<td>Bulk DNA/WES</td>
<td>WES</td>
<td>3</td>
<td>3</td>
<td>No Change to levels. Module name changed from &quot;Bulk DNA&quot; to &quot;WES&quot;</td>
</tr>
<tr>
<td>DigitalPathology (Phase 2 only)</td>
<td>H&amp;E</td>
<td>NA</td>
<td>level 2 only</td>
<td>DigitalPathology is a new Assay Module. In Phase 1, this data type would have been submitted as &quot;Imaging&quot;.</td>
</tr>
<tr>
<td>Imaging (Phase 1 only)</td>
<td>H&amp;E, CODEX, CyCIF, MIBI, mIF, MxIF, mIHC, IMC</td>
<td>4</td>
<td>NA</td>
<td>&quot;Imaging&quot; broken into two separate modules -- Digital Pathology and Multiplexed Tissue Imaging.</td>
</tr>
<tr>
<td>MultiplexedMicroscopy (Phase 2 only)</td>
<td>CODEX, CyCIF, MIBI, mIF, MxIF, mIHC, IMC, SABER</td>
<td>NA</td>
<td>levels 2-4 only (no level 1)</td>
<td>MultiplexedMicroscopy is a new Assay Module. In Phase 1, this data type would have been submitted as &quot;Imaging&quot;.</td>
</tr>
<tr>
<td>scRNA-seq</td>
<td>single cell or nuclei RNA-seq</td>
<td>4</td>
<td>3 total levels (1, 2, and level 3/4)</td>
<td>Level 3 and 4 combined into one level for submission of standardized h5ad files. See <doc-anchor-trigger to="#specific-assayassay-module-requirements">Specific Assay/Assy Module Requirements</doc-anchor-trigger> for more information.</td>
</tr>
<tr>
<td>Spatial Transcriptomics/SpatialOmics</td>
<td>10X Visium, 10X Xenium, Nanostring CosMX</td>
<td>platform-specific levels</td>
<td>1 required level (level 3, experiment bundle) with an accompanying spatial panel information file; optional levels 1 and 4</td>
<td>Change from platform-specific Spatial Transcriptomics models to a unified SpatialOmics model which can accomodate additional -omic data types. Note: level 3 bundles <strong>cannot contain controlled-access data</strong> such as FASTQs or BAMs.</td>
</tr>
</tbody>
</table>
</div>
</doc-panel>
<doc-anchor-target id="accepted-file-formats">
    <h2>
        <doc-anchor-trigger class="header-anchor-trigger" to="#accepted-file-formats">#</doc-anchor-trigger>
        <span>Accepted File Formats</span>
    </h2>
</doc-anchor-target>
<p>In HTAN Phase 2, there are specific requirements for file formats which are checked for each assay type. Please expand the panel below for more information.</p>
<doc-panel id="htan-phase-2-accepted-file-formats" collapsed>
    <template #title>HTAN Phase 2 Accepted File Formats</template>
<div class="table-wrapper scrollbar overflow-hidden">
<table class="comfortable">
<thead>
<tr>
<th>Assay Type</th>
<th>Level</th>
<th>Accepted File Types</th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td>DigitalPathology</td>
<td>2</td>
<td><a href="https://www.openmicroscopy.org/bio-formats/">Bio-Formats</a> or <a href="https://openslide.org/api/python/">openslide python</a> compatible format. (e.g. ome-tiff, tiff, qptiff, svs)</td>
<td></td>
</tr>
<tr>
<td>MultiplexedMicroscopy</td>
<td>2</td>
<td>ome-tiff preferred, <a href="https://www.openmicroscopy.org/bio-formats/">Bio-Formats</a> or <a href="https://openslide.org/api/python/">openslide python</a> compatible format accepted.</td>
<td></td>
</tr>
<tr>
<td></td>
<td>3</td>
<td>ome-tiff</td>
<td></td>
</tr>
<tr>
<td></td>
<td>4</td>
<td>csv, h5ad</td>
<td></td>
</tr>
<tr>
<td>scRNA-seq</td>
<td>1</td>
<td>fastq, fastq.gz</td>
<td></td>
</tr>
<tr>
<td></td>
<td>2</td>
<td>bam, cram</td>
<td></td>
</tr>
<tr>
<td></td>
<td>3/4</td>
<td>h5ad</td>
<td></td>
</tr>
<tr>
<td>SpatialOmics</td>
<td>1</td>
<td>gz, tar.gz</td>
<td></td>
</tr>
<tr>
<td></td>
<td>3</td>
<td>gz, tar.gz</td>
<td></td>
</tr>
<tr>
<td></td>
<td>Panel</td>
<td>csv</td>
<td></td>
</tr>
<tr>
<td></td>
<td>4</td>
<td>gz, tar.gz</td>
<td></td>
</tr>
<tr>
<td>WES</td>
<td>1</td>
<td>fastq, fastq.gz</td>
<td></td>
</tr>
<tr>
<td></td>
<td>2</td>
<td>bam, cram</td>
<td></td>
</tr>
<tr>
<td></td>
<td>3</td>
<td>vcf, vcf.gz</td>
<td></td>
</tr>
</tbody>
</table>
</div>
</doc-panel>
<doc-anchor-target id="specific-assayassay-module-requirements">
    <h2>
        <doc-anchor-trigger class="header-anchor-trigger" to="#specific-assayassay-module-requirements">#</doc-anchor-trigger>
        <span>Specific Assay/Assay Module Requirements</span>
    </h2>
</doc-anchor-target>
<doc-panel id="genomic-references-and-annotation-versions" collapsed>
    <template #title>Genomic References and Annotation Versions</template>
<doc-anchor-target id="genomic-references-and-annotation-versions">
    <h3>
        <doc-anchor-trigger class="header-anchor-trigger" to="#genomic-references-and-annotation-versions">#</doc-anchor-trigger>
        <span>Genomic References and Annotation Versions</span>
    </h3>
</doc-anchor-target>
<p>HTAN does not restrict data contributors to a specific genomic reference or annotation versions for most sequencing data with the exception of level 3/4 scRNA-seq data. Please see the <a href="../../data_submission/scrnaseq_data_submission/">Phase 2 Single Cell RNA-seq page</a> for more information regarding the scRNA-seq requirements.</p>
<p>However, HTAN data contributors must provide the genomics reference and annotation version used for any aligned sequencing files. This information can be found in the level 2 metadata.</p>
</doc-panel>
<doc-panel id="digital-pathology-and-multiplexmicroscopy" collapsed>
    <template #title>Digital Pathology and MultiplexMicroscopy</template>
<doc-anchor-target id="digital-pathology-and-multiplexmicroscopy">
    <h3>
        <doc-anchor-trigger class="header-anchor-trigger" to="#digital-pathology-and-multiplexmicroscopy">#</doc-anchor-trigger>
        <span>Digital Pathology and MultiplexMicroscopy</span>
    </h3>
</doc-anchor-target>
<p>HTAN can only accept <a href="../../data_submission/data_deidentification/">de-identified information</a>. Imaging equipment will often capture dates and other information and store this in the header of the resulting image file. <strong>HTAN cannot accept image files with any dates in the header</strong>. All image files must have dates or other identifying information removed from the image file header.</p>
</doc-panel>
<doc-panel id="scrna-seq" collapsed>
    <template #title>scRNA-seq</template>
<doc-anchor-target id="scrna-seq">
    <h3>
        <doc-anchor-trigger class="header-anchor-trigger" to="#scrna-seq">#</doc-anchor-trigger>
        <span>scRNA-seq</span>
    </h3>
</doc-anchor-target>
<p>Specific file requirements for single cell RNA-seq h5ad files are modeled after <a href="https://cellxgene.cziscience.com/docs/032__Contribute%20and%20Publish%20Data">CELLxGENE&#x27;s requirements</a>. Please see the <a href="../../data_submission/scrnaseq_data_submission/">Phase 2 Single Cell RNA-seq page</a> for more information.</p>
</doc-panel>

                                
                                <!-- Required only on API pages -->
                                <doc-toolbar-member-filter-no-results></doc-toolbar-member-filter-no-results>
                            </div>
                            <footer id="retype-content-footer" class="clear-both">
                            
                                <nav id="retype-nextprev" class="print:hidden flex mt-14">
                                    <div class="w-1/2">
                                        <a class="px-5 py-4 h-full flex items-center break-normal font-medium text-body-link border border-base-border hover:border-base-border-hover rounded-l-lg transition-colors duration-150 relative hover:z-5" href="../../data_model/metadata_standards/">
                                            <svg xmlns="http://www.w3.org/2000/svg" class="mr-3" width="24" height="24" viewBox="0 0 24 24" fill="currentColor" overflow="visible"><path d="M19 11H7.41l5.29-5.29a.996.996 0 10-1.41-1.41l-7 7a1 1 0 000 1.42l7 7a1.024 1.024 0 001.42-.01.996.996 0 000-1.41L7.41 13H19c.55 0 1-.45 1-1s-.45-1-1-1z" /><path fill="none" d="M0 0h24v24H0z" /></svg>
                                            <span>
                                                <span class="block text-xs font-normal text-base-text-muted">Previous</span>
                                                <span class="block mt-1">Metadata Standards</span>
                                            </span>
                                        </a>
                                    </div>
                            
                                    <div class="w-1/2">
                                        <a class="px-5 py-4 -mx-px h-full flex items-center justify-end break-normal font-medium text-body-link border border-base-border hover:border-base-border-hover rounded-r-lg transition-colors duration-150 relative hover:z-5" href="../../data_model/identifiers/">
                                            <span>
                                                <span class="block text-xs font-normal text-right text-base-text-muted">Next</span>
                                                <span class="block mt-1">Identifiers</span>
                                            </span>
                                            <svg xmlns="http://www.w3.org/2000/svg" class="ml-3" width="24" height="24" viewBox="0 0 24 24" fill="currentColor" overflow="visible"><path d="M19.92 12.38a1 1 0 00-.22-1.09l-7-7a.996.996 0 10-1.41 1.41l5.3 5.3H5c-.55 0-1 .45-1 1s.45 1 1 1h11.59l-5.29 5.29a.996.996 0 000 1.41c.19.2.44.3.7.3s.51-.1.71-.29l7-7c.09-.09.16-.21.21-.33z" /><path fill="none" d="M0 0h24v24H0z" /></svg>
                                        </a>
                                    </div>
                                </nav>
                            </footer>
                        </main>
                
                        <div id="retype-page-footer" class="print:border-none border-t border-base-border pt-6 mb-8">
                            <footer class="flex flex-wrap items-center justify-between print:justify-center">
                                <div id="retype-footer-links" class="print:hidden">
                                    <ul class="flex flex-wrap items-center text-sm">
                                    </ul>
                                </div>
                                <div id="retype-copyright" class="print:justify-center py-2 text-footer-text font-footer-link-weight text-sm leading-relaxed"></div>
                            </footer>
                        </div>
                    </div>
                
                    <!-- Rendered if sidebar right is enabled -->
                    <!-- Sidebar right skeleton-->
                    <div v-cloak class="fixed top-0 bottom-0 right-0 translate-x-full bg-sidebar-right-bg border-sidebar-right-border lg:sticky lg:border-l lg:shrink-0 lg:pt-6 lg:transform-none sm:w-1/2 lg:w-64 lg:z-0 md:w-104 sidebar-right skeleton">
                        <div class="pl-5">
                            <div class="w-32 h-3 mb-4 bg-skeleton-bg rounded-full loading"></div>
                            <div class="w-48 h-3 mb-4 bg-skeleton-bg rounded-full loading"></div>
                            <div class="w-40 h-3 mb-4 bg-skeleton-bg rounded-full loading"></div>
                        </div>
                    </div>
                
                    <!-- User should be able to hide sidebar right -->
                    <doc-sidebar-right v-cloak></doc-sidebar-right>
                </div>

            </div>
        </div>
    
        <doc-search-mobile></doc-search-mobile>
        <doc-back-to-top></doc-back-to-top>
    </div>


    <div id="retype-overlay-target"></div>

    <script data-cfasync="false">window.__DOCS__ = { "title": "File Standards", level: 2, icon: "file", hasPrism: false, hasMermaid: false, hasMath: false, tocDepth: 23 }</script>
</body>
</html>
